static/styles/portico/portico.css

```diff
@@ -1397,7 +1396,7 @@ input.new-organization-button {
             }
 
             + .hamburger {
-                transform: translateX(calc(100vw - 50px));
+                transform: translateX(calc(100vw - 72px));
```

Comment:
@vinitS101 as pointed by you. This was happening.
![image](https://user-images.githubusercontent.com/56730716/119663233-cdd64e80-be4f-11eb-9cf4-daaf834646c1.png)

This shifts a bit to left so that there will be no overlap in anycase.
![image](https://user-images.githubusercontent.com/56730716/119663328-e21a4b80-be4f-11eb-9d3b-7e72982c0d05.png)
Also on mobile devices, it looks fine and fixes both the problems.